src/Raven.Server/Web/Studio/LicenseHandler.cs

```diff
@@ -204,6 +207,74 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
                 }
             }
         }
+
+        [RavenAction("/studio/whats-new", "GET", AuthorizationStatus.ValidUser, EndpointType.Read)]
```

Comment:
It's put in `LicenseHandler` but it has `/studio`. Please try to be consistent and decide where this endpoint should be implemented. Please check other similar endpoints like that. Is it more about the license itself or more about exposing something specific for the studio? Also creating a dedicated handled is an option.

Reply:
Created separate `UpgradeInfoHandler`, new endpoint doesn't fit any of existing ones